### PR TITLE
Tweak: Remove the Folder icon when using a nested tab [ed-8314]

### PR DESCRIPTION
**Pull request by hein-obox**

### CODE REVIEW

File: tests/playwright/sanity/modules/tabs-v2/tabs.test.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+const { test, expect } = require( '@playwright/test' );
+const WpAdminPage = require( '../../../pages/wp-admin-page' );
+
+test.describe( 'Nested Tabs tests', () => {
+	test( 'Count the number of icons inside the Add Section element', async ( { page }, testInfo ) => {
+		// Arrange.
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		await wpAdmin.setExperiments( {
+			container: true,
+            'nested-elements': true,
```
Comment (davseve) [inline tests/playwright/sanity/modules/tabs-v2/tabs.test.js:10]: This object is not aligned well.

**Review by davseve — CHANGES_REQUESTED** [review]

File: tests/playwright/sanity/modules/tabs-v2/tabs.test.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+		} );
+
+		const editor = await wpAdmin.useElementorCleanPost(),
+			container = await editor.addElement( { elType: 'container' }, 'document' );
+
+		// Add widgets.
+		await editor.addWidget( 'tabs-v2', container );
+		await editor.getPreviewFrame().waitForSelector( '.elementor-tabs-content-wrapper .e-con.elementor-active' );
```
Comment (davseve) [inline tests/playwright/sanity/modules/tabs-v2/tabs.test.js:18]: Should we delete the page?

**Review by davseve — CHANGES_REQUESTED** [review]

File: tests/playwright/sanity/modules/tabs-v2/tabs.test.js (hunk 2 above)
Comment (hein-obox) [reply tests/playwright/sanity/modules/tabs-v2/tabs.test.js:18]: @davseve I don't think, but can you ask the question in the Playwright slack channel.

Comment (davseve) [reply tests/playwright/sanity/modules/tabs-v2/tabs.test.js:18]: According to what was said in the Playwright channel, we should return the site to the same state we received it.

**Review by davseve — CHANGES_REQUESTED** [review]

**Review by davseve — APPROVED** [review]

**Pull request merged**